containers/haskell/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Haskell (Community)",
+
+  // Update the 'dockerfile' property if you aren't using the standard 'Dockerfile' filename.
+  "build": {
+          "dockerfile": "Dockerfile"
+  },
+
+  // Set *default* container specific settings.json values on container create.
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash"
```

Comment:
This setting is now deprecated and has been removed from other container definitions in #860.